components/Overview.js

```diff
@@ -21,7 +21,7 @@ export default function Overview() {
     <ul>
       {shuffledPictures.map((picture) => (
         <StyledListItem key={picture.id}>
-          <FavoriteButton />
+          <FavoriteButton onToggleFavorite={onToggleFavorite} />
```

Comment:
You need to pass the information which image to toggle to the toggle function.
```suggestion
          <FavoriteButton onToggleFavorite={() => onToggleFavorite(picture.id)} />
```